modules/infra/control/gr_graph.h

```diff
@@ -53,4 +53,12 @@ extern struct node_infos node_infos;
 		STAILQ_INSERT_TAIL(&node_infos, &drop_info_##node_name, next);                     \
 	}
 
+#ifdef __GROUT_UNIT_TEST__
+#include <gr_cmocka.h>
+
+static inline void
+__wrap_rte_node_enqueue_x1(struct rte_graph *, struct rte_node *, rte_edge_t next, void *) {
+	check_expected(next);
+}
+#endif
```

Comment:
Since the #define needs to be in .c files, having this function here is confusing. Sorry about the double reversal but could you move it back next to `#define rte_node_enqueue_x1 __wrap_rte_node_enqueue_x1` in ip_input.c?